pylintrc.tests

```diff
@@ -0,0 +1,95 @@
+[MASTER]
```

Comment:
We need some different PyLint config for tests, that we wouldn't want to use for main code. I don't think a pylintrc file can inherit from another